src/foam/nanos/auth/email/EmailVerificationWebAgent.js

```diff
@@ -65,25 +63,36 @@ foam.CLASS({
         String             redirect         = request.getParameter("redirect");
         User               user             = (User) userDAO.find(Long.valueOf(userId));
 
+        TranslationService ts = (TranslationService) x.get("translationService");
+        String local = user.getLanguage().getCode().toString();
```

Comment:
Why is this required?

Reply:
Ah, ok, alternative is creating a subcontext with subject set that is used in call to translation service.  
this is fine.